examples/e2e/eks-static/terraform.template.tfvars

```diff
@@ -66,8 +66,3 @@ worker_azs_subnets_map = {
   "{enter AWS Availability Zone A}" = "{enter Subnet for AZ A, ...}"
   "{enter AWS Availability Zone B}" = "{enter Subnet for AZ A, ...}"
 }
-
- master_azs_subnets_map = {
```

Comment:
why this is deleted?

Reply:
It was causing `go generate` to fail because that key is already present in the same file.